fs/config/configfile/configfile_ssh.go

```diff
@@ -0,0 +1,91 @@
+package configfile
+
+import (
+	"fmt"
+	"github.com/kevinburke/ssh_config"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+var keyMapping = map[string]string{
+	"identityfile": "key_file",
+	"pubkeyfile":   "pubkey_file",
+	"user":         "user",
+	"hostname":     "host",
+	"port":         "port",
+	"password":     "pass",
+}
+
+func ConvertToIniKey(customKey string) string {
+	if iniKey, found := keyMapping[strings.ToLower(customKey)]; found {
+		return iniKey
+	}
+
+	return customKey
+}
+
+type SshConfig map[string]map[string]string
```

Comment:
one key can have multiple values

```
$ ssh -G example.com | grep ^identityfile
identityfile ~/.ssh/id_rsa
identityfile ~/.ssh/id_ecdsa
identityfile ~/.ssh/id_ecdsa_sk
identityfile ~/.ssh/id_ed25519
identityfile ~/.ssh/id_ed25519_sk
identityfile ~/.ssh/id_xmss
```

see also https://github.com/kevinburke/ssh_config

<blockquote>

Certain directives can occur multiple times for a host (such as `IdentityFile`),
so you should use the `GetAll` or `GetAllStrict` directive to retrieve those
instead.

```go
files := ssh_config.GetAll("myhost", "IdentityFile")
```

</blockquote>

Reply:
Thanks @milahu. I will consider that in the next iteration.
At the moment my biggest problem it, its writing the merged config into rclone config file. So maybe it would be better to move the ssh config into `backend/ftp` but there I experienced some problems, so this is just a draft atm.

Reply:
@milahu how would you provide more than one `key_file` using rclone sftp? I think it is not possible with the current rclone version.

Reply:
@milahu I would split that as well into another issue. So one for multiple `IndentityFile`'s and another one for default values. (question, can you think about more default values, that might be interesting to use with rclone?)

Again, @ncw this another reason I think parsing the ssh config into Env does not for me like decoupled enough... even though this would work. Now for getting this kind of feature would imply to parse all given ssh configs and default IdentityFile's into rclone env, without actually the need of all config that has been loaded into Envs.